packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -509,6 +510,48 @@ class MinimumTextContrastGuideline extends AccessibilityGuideline {
   String get description => 'Text contrast should follow WCAG guidelines';
 }
 
+/// A guideline which verifies that all nodes that contribute semantics via text
+/// meet **WCAG AAA** contrast levels.
+///
+/// The AAA level is defined by the Web Content Accessibility Guidelines:
+/// https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
+///
+/// This guideline enforces a stricter contrast ratio:
+///  * Normal text must have a contrast ratio of at least 7.0
+///  * Large or bold text must have a contrast ratio of at least 4.5
+///
+/// See also:
+///  * [MinimumTextContrastGuideline], which follows the WCAG AA level.
+///  * [AccessibilityGuideline], which provides an overview of guidelines.
+@visibleForTesting
+class MinimumTextContrastGuidelineAAA extends MinimumTextContrastGuideline {
+  /// Create a new [MinimumTextContrastGuidelineAAA].
+  const MinimumTextContrastGuidelineAAA();
+
+  /// The minimum contrast ratio for large text (bold ≥14px or ≥18px).
+  ///
+  /// Defined by WCAG AAA standard.
```

Comment:
Super nit: Would it be useful to link to the standard here? Since this will be its own page in the docs and people might not know what this refers to? Same for kAAAMinimumRatioNormalText below.